### PR TITLE
[1210] Add ucas_status on course model and expose on serializer

### PR DESCRIPTION
**Pull request by tvararu**
### Context

:shipit:

### CODE REVIEW

**Comment by tvararu** [comment]
[EPIC: Courses table in manage-courses-frontend](https://trello.com/c/Ju2Kfadq/1210-epic-courses-table-in-manage-courses-frontend)

File: app/models/course.rb [hunk 1]
```diff
@@ -124,4 +128,11 @@ def content_status
       :draft
     end
   end
+
+  def ucas_status
+    return :running if findable?
```
Comment (heidar) [inline app/models/course.rb]: How come a course is running if it's findable? Are they the same concept?

Comment (misaka) [reply app/models/course.rb]: Findable means it shows up on Find / Search and compare.
Running probably means it's on UCAS' Apply system.

Comment (tvararu) [reply app/models/course.rb]: @benilovj instructed this and can explain better I imagine 😬

**Pull request merged**